spectral/README.md

```diff
@@ -132,9 +132,9 @@ properties:
 
 **Severity**: error
 
-### rhoas-schema-name-camel-case
+### rhoas-schema-name-pascal-case
 
-All JSON schema objects defined in `components.schemas` must follow `CamelCase`.
+All JSON schema objects defined in `components.schemas` must follow `PascalCase`.
```

Comment:
I personally prefer to do more small PRs to get things like this nailed and merged. It makes you work faster and easier to review itself.

Reply:
I think you can pick what works for you. I like more PRs that are small to give you sense of completion. For large PRs we often end up with some blockers that dragging them. 

I tend to organize my work into smaller PRs intentionally to minimize amount of blockers. for example for this work I would have:

- PR for documentation and rule changes
- PR for removing unneeded rules
- PR for updating rules to v6 (current PR)
- PR for dev experience for building rules/doc

Etc.